### DISCUSSION TITLE
Easier Discord Tagging

### DISCUSSION THREAD

**Discussion by vee-the-plant** [RevertIT/mybb-rt_discord_webhooks#11] [Ideas]
I think that this would benefit my forum if we could tag without needing the embed; we don't want our posts showing on discord due to mature content and people having different triggers. Could we @ someone in the reply without that embed? And can the tagging for discord be made simpler to use the persons discord username, not their discord ID number?

**Comment by RevertIT** [comment]
> And can the tagging for discord be made simpler to use the persons discord username, not their discord ID number?

Unfortunately, no. It's the discord limitation for webhooks. We could theoretically make a map to `username <-> discord id`, but that map would be manually populated for each and every user.

> we don't want our posts showing on discord due to mature content and people having different triggers.

You should be able to select specific forums where to enable webhooks.
